packages/babel-plugin/src/styled/index.tsx

```diff
@@ -108,6 +108,56 @@ const extractStyledDataFromNode = (
   return undefined;
 };
 
+/**
```

Comment:
While this is an example I provided, it's probably not something we necessarily want to support since it is invalid syntax. We should get developers to write more straightforward code, even though this is supported by emotion and styled components. Having the logic here can be useful to provide a more meaningful error on how to fix the problem, otherwise we will need to codemod it usages exist in products. I haven't checked this, I presume there will be some.

cc @madou

Reply:
- [ ] Do not support use case, but throw error instead

Reply:
done in https://github.com/atlassian-labs/compiled/pull/788/commits/8a82bab6a28ab8206e2325ae777a5d574f6738ca